projects/packages/videopress/package.json

```diff
@@ -5,7 +5,7 @@
 	"description": "VideoPress package",
 	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/packages/videopress/#readme",
 	"bugs": {
-		"url": "https://github.com/Automattic/jetpack/labels/[Package] Videopress"
+		"url": "https://github.com/Automattic/jetpack/labels/[Package] VideoPress"
```

Comment:
I think the project structure linter (.github/files/lint-project-structure.sh IIRC) would need to be updated to allow mixed case. Right now, it just sees `videopress` as the folder so wants `Videopress`. @anomiex

Reply:
```suggestion
		"url": "https://github.com/Automattic/jetpack/labels/[Package] Videopress"
```

Reply:
Copying my reply from Slack to here to close the loop:

> You'd have to add a special case near [.github/files/lint-project-structure.sh#L153](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/.github/files/lint-project-structure.sh#L153) and [.github/files/lint-project-structure.sh#L306](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/.github/files/lint-project-structure.sh#L306) (maybe abstracting it into a function) to generate the correct URL, and also add one to the function at [projects/github-actions/repo-gardening/src/tasks/add-labels/index.js#L15](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/projects/github-actions/repo-gardening/src/tasks/add-labels/index.js#L15) to make it assign the correct label. Also possibly something near [projects/github-actions/repo-gardening/src/tasks/check-description/index.js#L118](https://github.com/Automattic/jetpack/blob/trunk/projects/github-actions/repo-gardening/src/tasks/check-description/index.js#L118) too. Maybe other places too.